packages/test/test-end-to-end-tests/src/test/gc/gcTrailingOps.spec.ts

```diff
@@ -124,7 +127,8 @@ describeCompat("GC trailing ops tests", "NoCompat", (getTestObjectProvider) => {
 			if (provider.driver.type !== "local") {
 				this.skip();
 			}
-			settings["Fluid.GarbageCollection.TestOverride.SweepTimeoutMs"] = sweepTimeoutMs;
+			settings["Fluid.GarbageCollection.TestOverride.TombstoneTimeoutMs"] =
```

Comment:
This doesn't seem right. The sweep timeout should be tombstone timeout + grace period.

Reply:
Wow...

Reply:
sweepGracePeriodMs is 0 so it works out, but I'll fix the code, good catch.